src/main/kotlin/no/nav/familie/ba/sak/behandling/resultat/BehandlingsresultatUtils.kt

```diff
@@ -13,28 +13,64 @@ import no.nav.fpsak.tidsserie.LocalDateTimeline
 
 object BehandlingsresultatUtils {
 
+    val ikkeStøttetFeil =
+            Feil(frontendFeilmelding = "Behandlingsresultatet du har fått på behandlingen er ikke støttet i løsningen enda. Ta kontakt med Team familie om du er uenig i resultatet.",
+                 message = "Behandlingsresultatet er ikke støttet i løsningen, se securelogger for resultatene som ble utledet.")
+
     fun utledBehandlingsresultatBasertPåYtelsePersoner(ytelsePersoner: List<YtelsePerson>): BehandlingResultat {
+        val (søknad, andre) = ytelsePersoner.partition { it.erFramstiltKravForINåværendeBehandling }
```

Comment:
```suggestion
        val (framstiltNå, framstiltTidligere) = ytelsePersoner.partition { it.erFramstiltKravForINåværendeBehandling }
```
Evt (fraSøknad, fraTidligere) hvis man vet det alltid er søknad der?

Reply:
haha, denne har kommet med som en glipp 😆  Var veldig WIP navn :)